### PR TITLE
Dev documentation login done

### LINKED ISSUE #92: Documentation for Login API
## Context
Login API is used to Login registered user 

## Problem
The Technical documentation of the API is needed 

## Solution 

- [ ] Create Documentation about how to setup the API (Overview of implementation details)
- [ ] Create Documentation about how to use the API (How to call the API )

### PR DESCRIPTION
**Pull request by jyaneshseeam**
# Pull Request name

Login API documentation

## Source 
Closes #92 

## Description 

Login API documentation

### Summary of PR  

Documentation created to facilitate understanding of API

### Purpose 

Documentation of code

## Review and Testing Steps
- [x] Pull this branch
- [x]  Check app\documentation\login.md for documentation of API
- [x]  Comment down if something is not clear

## Additional Information (If applicable)
Check app\documentation\login.md for documentation of API

### Notes
NA

### CODE REVIEW

**Review by ja-chisholm — APPROVED** [review]
Looks good, everything seems clear to me

File: app/documentation/login.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Emergency suppport feature documentation:
+
+- [ ] User clicks Sign in Button
+- [ ] User inputs user Name -> could be email or username itself
+- [ ] User inputs Password
+- [ ] User clicks Sign in
+- [ ] App sends data entered to backend to validate
+- [ ] Backend sends back login token
+- [ ] App saves login token in async storage
```
Comment (sontushkpaul) [inline app/documentation/login.md]: @jyaneshseeam @ja-chisholm @DemiAdisa 
Please do not use checklist other than issue. Use order or unordered list instead for PR and Documentation

- Item 1
- Item 2
- Item 3

or 
1. Item A
2. Item B
3. Item C

Checkbox is used to track to do list

Comment (sontushkpaul) [reply app/documentation/login.md]: @jyaneshseeam any progress?

Comment (jyaneshseeam) [reply app/documentation/login.md]: > @jyaneshseeam any progress?

just pushed new one, please approve

Comment (sontushkpaul) [reply app/documentation/login.md]: @jyaneshseeam lint errors are there

Comment (jyaneshseeam) [reply app/documentation/login.md]: > @jyaneshseeam lint errors are there

It's coming from demi's code in the front end, I mentioned that in the group.

Comment (jyaneshseeam) [reply app/documentation/login.md]: @sontushkpaul  I just edited the part causing the linting error in front end should be good now

**Review by sontushkpaul — APPROVED** [review]

**Pull request merged**